database/frontend/design-inspirations.json

```diff
@@ -1,15 +1,101 @@
-[{
-        "name": "Dribbble",
-        "description": "Explore the world’s leading design portfolios.",
-        "url": "https://dribbble.com",
-        "category": "frontend",
-        "subcategory": "design-inspiration"
-    },
-    {
-        "name": "Godly Website",
-        "description": "A curated collection of good web design from all over the web",
-        "url": "https://godly.website",
-        "category": "frontend",
-        "subcategory": "design-inspiration"
-    }
-]
\ No newline at end of file
+[
+  {
+    "name": "Dribbble",
+    "description": "Explore the world’s leading design portfolios.",
+    "url": "https://dribbble.com",
+    "category": "frontend",
+    "subcategory": "design-inspiration"
+  },
+  {
+    "name": "Godly Website",
+    "description": "A curated collection of good web design from all over the web",
+    "url": "https://godly.website",
+    "category": "frontend",
+    "subcategory": "design-inspiration"
+  },
+  {
+    "name": "Behance",
+    "description": "Behance is the world's largest creative network for showcasing and discovering creative work.",
+    "url": "https://www.behance.net/",
+    "category": "frontend",
+    "subcategory": "design-inspiration"
+  },
+  {
+    "name": "Refero Design",
+    "description": "Get inspired by the best examples of product design.",
+    "url": "https://refero.design/",
+    "category": "frontend",
+    "subcategory": "design-inspiration"
+  },
+  {
+    "name": "Page Collective",
+    "description": "Get inspired and keep up with the latest landing page design trends.",
+    "url": "https://pagecollective.com/",
+    "category": "frontend",
+    "subcategory": "design-inspiration"
+  },
+  {
+    "name": "Landingfolio",
+    "description": "Landingfolio features the best landing page designs, templates, component and more on teh web.",
+    "url": "https://www.landingfolio.com/",
+    "category": "frontend",
+    "subcategory": "design-inspiration"
+  },
+  {
+    "name": "Land-book",
+    "description": "Find the best hand-picked website design inspiration.",
```

Comment:
This could use a bit more detail. What makes Land-book distinctive from websites like Carrd or Behance?